micro-rdk/src/native/entry.rs

```diff
@@ -100,7 +100,7 @@ pub async fn serve_web_inner(
         exec.clone(),
     ));
 
-    let mut srv = ViamServerBuilder::new(mdns, cloned_exec, client_connector, app_config)
+    let mut srv = ViamServerBuilder::new(mdns, cloned_exec, client_connector, app_config, 3)
```

Comment:
Below, does it matter that the ESP32 and native `serve_web` functions have diverged, since the ESP32 one now takes the maximum as a parameter and the native one does not?

Reply:
I think its fine as long as we don't have a use case for native beyond testing we don't need to polish it too much